styles/stylesheet.css

```diff
@@ -0,0 +1,378 @@
+* {
+    margin: 0;
```

Comment:
Another thing good to add in this block is the `box-sizing: border box;` Google it to see how it works.